support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx

```diff
@@ -31,13 +38,41 @@ import {
 	sendTrackingEventsOnView,
 } from 'helpers/productPrice/subscriptions';
 import type { OphanComponentType } from 'helpers/tracking/trackingOphan';
-import { getOrigin, getQueryParameter } from 'helpers/urls/url';
+import {
+	addQueryParamsToURL,
+	getOrigin,
+	getQueryParameter,
+} from 'helpers/urls/url';
 import Prices from './content/prices';
 
+const countryPath = (countryGroupId: CountryGroupId) =>
+	countryGroups[countryGroupId].supportInternationalisationId;
+
 const getCheckoutUrl = (
+	countryId: IsoCountry,
 	billingPeriod: WeeklyBillingPeriod,
+	abParticipations: Participations,
 	orderIsGift: boolean,
+	promotion?: Promotion,
 ): string => {
+	// Gifting will be supported last
+	if (
+		abParticipations.guardianWeeklyGenericCheckout === 'variant' &&
+		!orderIsGift
+	) {
+		const countryGroupId = CountryGroup.fromCountry(countryId) ?? GBPCountries;
+		const productGuardianWeekly =
+			countryGroupId === 'GBPCountries'
+				? 'GuardianWeeklyDomestic'
+				: 'GuardianWeeklyRestOfWorld';
```

Comment:
Is this logic correct? I might be getting confused but I think @rupertbates mentioned that users in specific regions (US/AU/NZ/UK/CA/EU) get the `GuardianWeeklyDomestic` product, but the rest of the world get `GuardianWeeklyRestOfWorld`. The product catalog has prices for the regions I mentioned for the `GuardianWeeklyDomestic` product.

Reply:
Good spot, have rewritten using an existing call that operates as you mention  (`internationaliseProductAndRatePlan`), updated.